tui/src/confirmation.rs

```diff
@@ -84,6 +84,21 @@ impl FloatContent for ConfirmPrompt {
         frame.render_widget(List::new(paths_text), inner_area);
     }
 
+    fn handle_mouse_event(&mut self, event: &MouseEvent) -> bool {
+        match event.kind {
+            MouseEventKind::ScrollDown => {
+                self.scroll_down();
+                ConfirmStatus::None
+            }
+            MouseEventKind::ScrollUp => {
+                self.scroll_up();
+                ConfirmStatus::None
+            }
+            _ => ConfirmStatus::None,
+        };
```

Comment:
```suggestion
        match event.kind {
            MouseEventKind::ScrollDown => {
                self.scroll_down();
            }
            MouseEventKind::ScrollUp => {
                self.scroll_up();
            }
            _ => {}
        }
```
Why is this match statement returning an unused ConfirmStatus?

Reply:
I was wondering the same with handle_key_event